docs/reference/changelog-r2023.md

```diff
@@ -44,6 +44,7 @@ Released on ??
     - Added `proto_formatter.py` script to automatically indent PROTO files ([#6167](https://github.com/cyberbotics/webots/pull/6167)).
     - Disabled the computation of shadow reception on meshes when shadows are globally disabled([#6201](https://github.com/cyberbotics/webots/pull/6201)).
   - Bug Fixes
+    - Fixed random crashes while creating [Skin](skin.md) and Mesh(mesh.md) nodes ([#6218](https://github.com/cyberbotics/webots/pull/6218)).
```

Comment:
The link to the Mesh node is wrong because the squared brackets are missing, but can be fixed in the Blog PR.

Reply:
Oops, sorry.

Reply:
Done in 222bb6176f32106de0d4693681fd52a9af4d6d40.